cameo/strain_design/deterministic/linear_programming.py

```diff
@@ -130,11 +131,11 @@ def _reduce_to_nullspace(self, reactions):
     def _build_problem(self, essential_reactions, use_nullspace_simplification):
         logger.debug("Starting to formulate OptKnock problem")
 
-        self.essential_reactions = find_essential_reactions(self._model) + self._model.exchanges
+        self.essential_reactions = find_essential_reactions(self._model).union(self._model.exchanges)
```

Comment:
No, `essential_reactions` is not a method of cobra `Model` didn't really see the need for it there..